tests/test_rabfe_topology.py

```diff
@@ -139,6 +137,7 @@ def test_dual_topology_standard_decoupling():
     assert isinstance(nonbonded_potential, potentials.NonbondedInterpolated)
 
     expected_qlj = topology.standard_qlj_typer(mol_c)
+    expected_qlj[:, 0] = expected_qlj[:, 0] / 2  # charges should be halved
```

Comment:
Doesn't change anything since the charges are 0 in the test case, but seemed best to cover the example